shared/constants/fs.js

```diff
@@ -1061,7 +1048,7 @@ export const makeActionsForDestinationPickerOpen = (
     index,
     path,
   }),
-  flags.useNewRouter || !routePath
+  !routePath
```

Comment:
This seems wrong. Should just take the thing after `?`

Reply:
so the existing code is wrong is what you're saying?

Reply:
No existing code is right. Note that's a `||` not `&&`, so with nav2 we never hit the `:` case.